Detections/SecurityEvent/RDP_MultipleConnectionsFromSingleSystem.yaml

```diff
@@ -69,23 +69,26 @@ query: |
   // Where the ratio of today to previous 7 days is more than double.
   | where Ratio > threshold
   | project StartTimeUtc, EndTimeUtc, Account, IpAddress, ComputerSet, ComputerCountToday, ComputerCountPrev7Days, Ratio, AccountType, Activity, LogonTypeName, ProcessSet
-  | extend timestamp = StartTimeUtc, AccountCustomEntity = Account, IPCustomEntity = IpAddress
+  | extend timestamp = StartTimeUtc
+  | extend AccountName = tostring(split(Account, "@")[0]), AccountUPNSuffix = tostring(split(Account, "@")[1])
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: Name
```

Comment:
Account is actually a Name + NTDomain in this case.  So you can map it, but you have to split it as follows:

| extend AccountName = tostring(split(Account, "\\")[1]), AccountNTDomain = tostring(split(Account, "\\")[0])